controllers/childrenController.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../models");
+
+// Defining methods for the childController
+module.exports = {
+  //find all children related to requestor
+  findAllByChild: function(req, res) {
+    db.Job
+      .findAll({where:{AssignedToEmail:req.body.email}})
```

Comment:
can you put some spacing  .findAll({where: {AssignedToEmail: req.body.email }}).